.gitignore

```diff
@@ -49,7 +49,10 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
+
+# great expectations examples
 examples/great_expectations/great_expectations/
+yellow_tripdata_sample_2019-01.csv
```

Comment:
what about just ignore all `*.csv` files? instead of a specific file name

Reply:
no b/c test data wouldn't be uploaded unless we wanted it specifically in this location

Reply:
because we have [csv files elsewhere](https://github.com/capitalone/DataProfiler/tree/main/dataprofiler/tests/data/csv) and we might want to add more in the future

Reply:
+1 on above... are the directions to download into the `examples/great_expectations/great_expectations/` folder?

Reply:
`examples/great_expectations/great_expectations/` this is created by gx, but I could see a `data` folder being in `examples/great_expectations/` and we could ignore all in that if we DL it to there.

Reply:
The profile pkl file could be saved there too.

Reply:
i created a `data` folder in the `examples/great_expectations/great_expectations/` dir and it's empty. I have instruction in the notebook

> Download the csv that contains the data we will be using for this example which can be found [here](https://github.com/great-expectations/gx_tutorials/blob/main/data/yellow_tripdata_sample_2019-01.csv) then save it to the `/examples/great_expectations/data` directory.

Reply:
In the examples, this also where the pkl file is created and save to as well

Reply:
dope -- so then in the `gitignore` we can just ignore `/examples/great_expectations/data`?

Reply:
I just pushed a change to ignore all folders in ge examples